namespaces/live-1.cloud-platform.service.justice.gov.uk/calculate-release-dates-api-dev/00-namespace.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: calculate-release-dates-api
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
```

Comment:
Need to be careful with env names - sometimes referred to as `development` and sometimes `dev`. I chose to use `dev` consistently throughout all the files - you can use `development` so long as its consistent.